src/main/kotlin/com/mparticle/kits/AppboyKit.kt

```diff
@@ -613,8 +613,18 @@ open class AppboyKit : KitIntegration(), AttributeListener, CommerceListener,
             }
         }
 
+        var sanitizedProductName: String = product.sku
+        try {
+            if (settings[FORWARD_SKU_AS_PRODUCT_NAME] == "True") {
+                sanitizedProductName = product.name
```

Comment:
Isn't this logic backwards? The setting is called "FORWARD_SKU_AS_PRODUCT_NAME" so when it's true shouldn't we be sending the `product.sku` and when false send `product.name`? Or is the setting just confusingly named?

Reply:
Never mind, looks like we just inherited a poorly named setting.